docs/wal.rst

```diff
@@ -37,7 +37,7 @@ WAL operations can only be performed by admin users via ``_system`` database.
     # List WAL transactions.
     wal.transactions()
 
-    # Flush WAL with garbage collection.
+    # Flush WAL with garbage relation.
```

Comment:
I suspect that there is still a "collection" 8))

Reply:
Ooops) Thanks for noticing! Fixed